src/scss/includes/components/panel.scss

```diff
@@ -12,14 +14,19 @@
   @include panel_radius();
   background-color: white;
   width: $panel_width;
-
-  &-header {
-    padding: 0 15px;
-    min-height: 40px;
-    color: #5c6f84;
-    span {
-      width: 100%;
-    }
+  display: flex;
+  flex-direction: column;
+
+  &-drawer {
```

Comment:
It needs a `flex-shrink: 0` too.
This would solve it being actually 13px and too close to the panel titles below. This removes the need for a top margin on the Favorites title.